rj_param_utils/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ find_package(rclcpp REQUIRED)
 # ======================================================================
 # Define Targets
 # ======================================================================
-add_library(rj_param_utils SHARED)
+add_executable(global_params)
```

Comment:
We can still keep the old target. We still need to link to it from other nodes so we can actually use the parameter library.